### ISSUE TITLE
withTheme

### ISSUE THREAD

**Issue by krawalli** [atanasster/grommet-controls#3]
with current grommet V2 all

~~~
import { withTheme } from 'grommet/components/hocs';
~~~
failed because

~~~
Attempted import error: 'withTheme' is not exported from 'grommet/es6/components/hocs'.
~~~

Seems it got a new home in
~~~
import { withTheme } from 'styled-components';
~~~

**Comment by atanasster** [comment]
Sorry about that, but the grommet team are just clueless about any best practices and break code as their whim.
I will update grommet-controls as soon as I have some time, in the meantime please use the previous grommet version.

**Comment by alansouzati** [comment]
we don't expose our internal `hocs`. it is not in your index.js.

If we do `import { withTheme } from "grommet";` you get undefined.

That said, we did not interpret this as a breaking change. I'm going to add a proxy to the styled-components one (for now), but please avoid importing anything that is not in our main index.js export as your library can break in unexpected ways. 

It is impossible for us to search through all libraries consuming grommet which path they are leveraging internally. And we cannot release a breaking change for every internal structural changes we do.

Also, I've mentioned in the slack channel and I'm relaying here. Please be more constructive when providing feedback. This sort of message can affect everyone around our community. We want to make sure to have a healthy environment where people can say/ask anything without being afraid to be attacked like this.

More love please ❤️

**Comment by atanasster** [comment]
Alan - please do not add fuel to the fire:
1. withTheme was the standard used at the time the grommet-controls was created
2. You do not have to search through many libraries- grommet-controls is the one major library using v2 of grommet.

**Comment by atanasster** [comment]
Fixed with grommet 2.2.1
